scripts/dev/contexts/variables/om80

```diff
@@ -19,3 +19,5 @@ export AGENT_IMAGE="${MDB_AGENT_IMAGE_REPOSITORY}:${AGENT_VERSION}"
 export CUSTOM_APPDB_VERSION=8.0.6-ent
 export TEST_MODE=opsmanager
 export OPS_MANAGER_REGISTRY="${BASE_REPO_URL}"
+
+export ops_manager_version="${CUSTOM_OM_VERSION}"
```

Comment:
why only here? Was it missing on the other variables? could we just rely on CUSTOM_OM_VERSION everywhere instead and remove ops_manager_version everywhere else instead?